lib/l10n/app_en.arb

```diff
@@ -1,301 +1,430 @@
+    "p_rename_will_change_account_displayed": "This will change how the account is displayed in the list.",
+    "l_delete_account": "Delete account",
+    "l_account_deleted": "Account deleted",
+    "p_warning_delete_account": "Warning! This action will delete the account from your YubiKey.",
+    "p_warning_disable_credential": "You will no longer be able to generate OTPs for this account. Make sure to first disable this credential from the website to avoid being locked out of your account.",
+    "l_account_name": "Account name",
+    "l_search_accounts": "Search accounts",
+    "l_accounts_used": "{used} of {capacity} accounts used",
+    "@l_accounts_used" : {
+        "placeholders": {
+            "used": {},
+            "capacity": {}
+        }
+    },
+    "l_num_digits": "{num} digits",
+    "@l_num_digits" : {
+        "placeholders": {
+            "num": {}
+        }
+    },
+    "l_num_sec": "{num} sec",
+    "@l_num_sec" : {
+        "placeholders": {
+            "num": {}
+        }
+    },
+    "l_duplicate_name": "This name already exists for the Issuer",
+    "l_issuer_optional": "Issuer (optional)",
 
+    "@_fido_credentials": {},
+    "l_credential": "Credential: {label}",
+    "@l_credential" : {
+        "placeholders": {
+            "label": {}
+        }
+    },
+    "w_credentials": "Credentials",
+    "l_ready_to_use": "Ready to use",
+    "l_register_sk_on_websites": "Register as a Security Key on websites",
+    "l_no_discoverable_accounts": "No discoverable accounts",
+    "l_delete_credential": "Delete credential",
+    "l_credential_deleted": "Credential deleted",
+    "p_warning_delete_credential": "This will delete the credential from your YubiKey.",
 
-    "appFailurePage_btn_unlock": "Unlock",
-    "appFailurePage_txt_info": "WebAuthn management requires elevated privileges.",
-    "appFailurePage_msg_permission": "Elevating permissions\u2026",
+    "@_fingerprints": {},
+    "l_fingerprint": "Fingerprint: {label}",
+    "@l_fingerprint" : {
+        "placeholders": {
+            "label": {}
+        }
+    },
+    "w_fingerprints": "Fingerprints",
+    "l_fingerprint_captured": "Fingerprint captured successfully!",
+    "l_fingerprint_added": "Fingerprint added",
+    "l_setting_name_failed": "Error setting name: {message}",
+    "@l_setting_name_failed" : {
+        "placeholders": {
+            "message": {}
+        }
+    },
+    "l_add_fingerprint": "Add fingerprint",
+    "l_fp_step_1_capture": "Step 1/2: Capture fingerprint",
+    "l_fp_step_2_name": "Step 2/2: Name fingerprint",
+    "l_delete_fingerprint": "Delete fingerprint",
+    "l_fingerprint_deleted": "Fingerprint deleted",
+    "l_warning_delete_fingerprint": "This will delete the fingerprint from your YubiKey.",
+    "l_no_fingerprints": "No fingerprints",
+    "l_set_pin_fingerprints": "Set a PIN to register fingerprints",
+    "l_no_fps_added": "No fingerprints have been added",
+    "l_fingerprint_renamed": "Fingerprint renamed",
```

Comment:
the naming of the keys for `fingerprint` sometimes uses `fp`/`fps`. I think it would be nice to unify this, so one can easily find all fingerprint related UI strings by search for a one string (either use only fp or only fingerprint)